addOns/grpc/grpc.gradle.kts

```diff
@@ -6,6 +6,18 @@ zapAddOn {
     manifest {
         author.set("ZAP Dev Team")
         url.set("https://www.zaproxy.org/docs/desktop/addons/grpc-support/")
+        extensions {
+            register("org.zaproxy.zap.extension.grpc.internal.ExtensionGrpcWebSocket") {
+                classnames {
+                    allowed.set(listOf("org.zaproxy.zap.extension.grpc.internal"))
```

Comment:
`org.zaproxy.addon.grpc.websocket`

Reply:
can you explain me this extension configuration why we are using org.zaproxy.addon.grpc.internal.websocket as there is no file or package as websocket inside internal folder and what register and classnames configuration means ?

Reply:
It should have, if the class was updated accordingly.

That's the package whose classes this optional extension is allowed to (un)load directly, since this is an optional extension it can be dynamically (un)loaded when the other add-on is (un)installed.

Reply:
ok thank you now its working.